.golangci.yml

```diff
@@ -44,7 +44,6 @@ linters-settings:
       - dupImport # https://github.com/go-critic/go-critic/issues/845
       - ifElseChain
       - octalLiteral
-      - whyNoLint
```

Comment:
Unfortunately, this is needed for latest v1.22.x releases due to go-critic update. The GitHub check should pass once https://github.com/go-critic/go-critic/issues/845 is merged and deployed.

Reply:
Any suggestion what should I do now?
1. Return the `whyNoLint` config
2. ???

Reply:
Just undo the `whyNoLint` removal commit and it should be good to merge.

Reply:
GolangCI failure is expected, yes.

To undo this commit you can do the following:
```
git reset --hard 6fd38300fbdc897e6786e899717e712cbecde0b8
git push -f uudashr 
```
Where 6fd38300fbdc897e6786e899717e712cbecde0b8 is your first commit, and `uudashr` is your fork remote name.

Reply:
Reverted

Reply:
Hi @tpounds already undo the `whyNoLint`, should good now